tools/backup-setup/config/backup.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: ark.heptio.com/v1
+kind: Backup
+metadata:
+  name: kyma-backup
+  namespace: heptio-ark
+spec:
+  includedNamespaces:
+  - production
+  excludedNamespaces:
+  - default
```

Comment:
Here are some namespaces missing like istio-system, kyma-integration. Just start a kyma instance and check which namespaces are created and not visible for the user.